sbt-plugin/src/main/scala/play/soap/sbtplugin/PlayClientGenerator.scala

```diff
@@ -39,7 +39,7 @@ class PlayClientGenerator extends ServiceGenerator with PlayGenerator {
     parseOutputName(packageName, filename, ".scala")
   }
 
-  private object PortMethodNameGenerator {
+  private[sbtplugin] object PortMethodNameGenerator {
```

Comment:
If this is only `private` then (it seems) in newer Scala versions the compiler optimizes away this private object (or at least the unused `transform` method inside it).
So without this fix the scripted tests fail with following error:
```
[info] [error] /tmp/sbt_1af24220/target/wsdl/main/sources/play/soap/testservice/HelloWorldService.scala:21:18: '=' expected but '.' found.
[info] [error]   def $portMethod.transform($port)(handlers: Handler[_ <: MessageContext]*): HelloWorld = {
[info] [error]                  ^
```
As you see `$portMethod.transform` should actually be the method call, however since the method "does not exist" the code will not be replaced.